src/components/Book.js

```diff
@@ -1,23 +1,43 @@
-import React from 'react';
+// /*eslint-disable */
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import Remove from './removeButton';
+import './Book.css';
+import { removeBook } from '../redux/books/books';
 
-const Book = (props) => {
-  const { books } = props;
+function Book({ id, title, author }) {
+  const dispatch = useDispatch();
+
+  const removeBooks = () => {
```

Comment:
Great job @Mukumbuta 🤝

- [ ] Kindly make sure the delete feature is working properly so that a book can be removed from the list.